src/rp2_common/pico_double/include/pico/double.h

```diff
@@ -16,64 +16,195 @@ extern "C" {
 #endif
 
 /** \file double.h
-*  \defgroup pico_double pico_double
+* \defgroup pico_double pico_double
 *
 * \brief Optimized double-precision floating point functions
 *
-* (Replacement) optimized implementations are provided of the following compiler built-ins
-* and math library functions:
+* An application can take control of the floating point routines used in the application over and above what is provided by the compiler,
+* by depending on the pico_double library. A user might want to do this
```

Comment:
nit: Should probably be a colon at the end of this line? (and same in `float.h`)